frontend/src/constants/languages.ts

```diff
@@ -71,3 +73,16 @@ export const languages = [
   "Vietnamese",
   "Xinhui"
 ]
+
+const languageByLowerCaseName: { [lowerCase: string]: string } = {}
+
+for (const lang of languages) {
+  languageByLowerCaseName[lang.toLowerCase()] = lang
+}
+
+export function fixLanguageName<T extends { languageName: string }>(toFix: T): T {
+  return {
+    ...toFix, 
+    languageName: languageByLowerCaseName[toFix.languageName.toLowerCase()] || toFix.languageName
```

Comment:
I'm wondering if we might consider always dealing with lowercase languages (if that's what's in the db) and use css to style the text nodes with capitalization in the various select elements that use them. possible enhancement for future. I suppose what would not address capitalizing CART and ASL, I wonder if we should just run a db update. anywho, works for. now, thanks for dealing w this.

Reply:
The languages are all hardcoded in a list somewhere anyway, which is what this code references. I figured once that is fixed it will naturally cause this code to be updated to something less hard-coded.